internal/pkg/skuba/upgrade/addon/versions.go

```diff
@@ -38,20 +38,23 @@ func UpdatedAddons(client clientset.Interface, clusterVersion *version.Version)
 	if err != nil {
 		return AddonVersionInfoUpdate{}, err
 	}
+	return UpdatedAddonsForAddonsVersion(clusterVersion, skubaConfig.AddonsVersion), nil
+}
+
+func UpdatedAddonsForAddonsVersion(clusterVersion *version.Version, addonsVersion kubernetes.AddonsVersion) AddonVersionInfoUpdate {
```

Comment:
see the lint job failing, it's possible that not commenting exported functions like this is triggering lint failures

Reply:
Yeah, the PR is not ready yet, I want to check if I can refactor some test code and finish the last details :)